### PR TITLE
Update autoloaders

### PR DESCRIPTION
**Pull request by rullzer**
Signed-off-by: Roeland Jago Douma <roeland@famdouma.nl>

### CODE REVIEW

**Review by nickvergessen — APPROVED** [review]

**Review by MorrisJobke — APPROVED** [review]

File: lib/composer/composer/autoload_classmap.php [hunk 1]
```diff
@@ -686,6 +686,7 @@
     'OC\\Core\\Migrations\\Version15000Date20180926101451' => $baseDir . '/core/Migrations/Version15000Date20180926101451.php',
     'OC\\Core\\Migrations\\Version15000Date20181015062942' => $baseDir . '/core/Migrations/Version15000Date20181015062942.php',
     'OC\\Core\\Migrations\\Version15000Date20181029084625' => $baseDir . '/core/Migrations/Version15000Date20181029084625.php',
+    'OC\\Core\\Migrations\\Version16000Date20190208142225' => $baseDir . '/core/Migrations/Version16000Date20190208142225.php',
```
Comment (MorrisJobke) [inline lib/composer/composer/autoload_classmap.php]: For me it says that this file is not present 🙈

Comment (MorrisJobke) [reply lib/composer/composer/autoload_classmap.php]: Locally and on CI.

Comment (rullzer) [reply lib/composer/composer/autoload_classmap.php]: Mmmm

Comment (rullzer) [reply lib/composer/composer/autoload_classmap.php]: Ah was not added to git yet so not gone when I stashed.

**Pull request merged**